src/platform/BUILD.gn

```diff
@@ -271,6 +271,11 @@ if (chip_device_platform != "none" && chip_device_platform != "external") {
       defines += [
         "CHIP_DEVICE_LAYER_TARGET_BEKEN=1",
         "CHIP_DEVICE_LAYER_TARGET=Beken",
+      ]
+    } else if (chip_device_platform == "mt793x") {
+      defines += [
+        "CHIP_DEVICE_LAYER_TARGET_MT793X=1",
+        "CHIP_DEVICE_LAYER_TARGET=mt793x",
         "CHIP_DEVICE_CONFIG_ENABLE_WIFI=${chip_enable_wifi}",
```

Comment:
You need add a line of "CHIP_DEVICE_CONFIG_ENABLE_WIFI=${chip_enable_wifi}". Now Beken's config is error, please fix it.